deferred-iterator.js

```diff
@@ -13,15 +12,14 @@ function DeferredIterator (options) {
 util.inherits(DeferredIterator, AbstractIterator)
 
 DeferredIterator.prototype.setDb = function (db) {
-  var it = this._iterator = db.iterator(this._options)
+  var it = (this._iterator = db.iterator(this._options))
```

Comment:
This was probably added by `prettier`? The parentheses are a bit weird

Reply:
Agreed! Fixing.